docs/cosmology/traits.rst

```diff
@@ -0,0 +1,54 @@
+.. _astropy-cosmology-traits:
+
+****************
+Cosmology Traits
+****************
+
+.. currentmodule:: astropy.cosmology.traits
+
+The ``traits`` module hosts various parts of cosmologies, such as the
+:class:`~astropy.cosmology.traits.ScaleFactor` or :class:`~astropy.cosmology.traits.TemperatureCMB`. These traits can be used to more easily
```

Comment:
It would be good to link to the glossary entry (there should be a glossary entry).

As a general remark, very long lines should be avoided. On GitHub I can comment individual lines, so the longer the line, the less clear it is what exactly am I commenting on.

Reply:
Done!